step.yml

```diff
@@ -225,8 +225,12 @@ inputs:
       is_sensitive: true
   - generate_universal_apk_if_none: "true"
     opts:
-      title: "Force generate universal APK for AABs"
+      title: "Force generate universal APK for AABs (DEPRECATED)"
       description: |
+        Universal APK generation is going to be  **DEPRECATED** with the next
```

Comment:
Universal APK generation is scheduled to be **DEPRECATED** in the next major release.